apps/a1/Dockerfile

```
@@ -0,0 +1,33 @@
+# Sorta hacky solution to being able to write to /workspace
+USER root
+
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /usr/local/bin/uv
+
+WORKDIR /workspace
+
+ENV UV_COMPILE_BYTECODE=1 \
+    UV_LINK_MODE=copy \
+    UV_NO_MANAGED_PYTHON=1
+
+COPY pyproject.toml uv.lock .
```

Hadolint (2.12.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)